src/pixbuf.c

```diff
@@ -28,13 +28,13 @@ void pixbuf_save_state_to_folder(GdkPixbuf *pixbuf, char *folder,
   char *c_time_string;
   char filename[255];
   char path[MAX_PATH];
-  size_t bytes_formated;
+  size_t bytes_formatted;
 
   c_time_string = ctime(&current_time);
   c_time_string[strlen(c_time_string) - 1] = '\0';
-  bytes_formated = strftime(filename, sizeof(filename), filename_format,
+  bytes_formatted = strftime(filename, sizeof(filename), filename_format,
```

Comment:
@kianmeng the clang-format is now failing here. See CI logs. Can you fix it please?

Reply:
@jtheoof Changes reverted.